src/Control/Email/Mailer.php

```diff
@@ -2,10 +2,10 @@
 
 namespace SilverStripe\Control\Email;
 
-use InvalidArgumentException;
-use SilverStripe\Control\HTTP;
-use SilverStripe\Core\Convert;
+use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Core\Object;
+use Swift_Mailer;
+use Swift_MailTransport;
```

Comment:
There’s a lot of Swift-specific code in this class, which shouldn’t really be inherited by modules that provide other mailers. Would it be better to make `Mailer` an interface?

```php
interface Mailer
{
    public function send($message);
}

class SwiftMailer implements Mailer
{
    public function send($message)
    {
        $swiftMessage = $message->getSwiftMessage();
        return $this->sendSwift($swiftMessage);
    }
}

// Email.php
Injector::inst()->get('SilverStripe\Email\Mailer')->send($this);
```